ui/README.md

```
@@ -134,30 +134,26 @@ The UI uses Redux Toolkit + RTK Query for state management and API communication
-  try {
-    await createProvider({
-      provider: 'openai',
-      keys: [{ value: 'sk-...', models: ['gpt-4'], weight: 1 }],
-      // ... other config
-    }).unwrap()
-    // Success handling
-  } catch (error) {
-    console.error(getErrorMessage(error))
-  }
-}
+	try {
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)